ddtrace/contrib/mysql/__init__.py

```diff
@@ -26,6 +26,17 @@
 """
 from ..util import require_modules
 
+# check `MySQL-python` availability
+required_modules = ['_mysql']
+
+with require_modules(required_modules) as missing_modules:
+    if not missing_modules:
+        # MySQL python package is not supported at the moment;
+        # here we raise an import error so that the external
+        # loader knows that the integration is not available
+        raise ImportError('No module named mysql-python')
```

Comment:
The error must be an `ImportError` because it's like if we're trying to import a module that is not here. However, the message may be different?